DataRepo/templates/upload.html

```diff
@@ -1,50 +1,109 @@
 {% extends "base.html" %}
 {% load static %}
 {% block content %}
-<div>
-    <h4>Animal, Sample, &amp; Accucor Data Submission Process</h4>
-    <hr />
+    <div>
+
+        <h4 id="animal-sample-accucor-data-submission-process">Animal, Sample, &amp; Accucor Data Submission Process</h4>
+        <h5 id="create-a-sample-sheet">Create a Sample Sheet</h5>
+        <p>An important part of TraceBase is collecting and maintaining a consistent set of metadata for each sample.</p>
+        <ol>
+            <li>Make a copy of the <a href="https://docs.google.com/spreadsheets/d/1To3495KxJkAtnAD9KVdzc162zKbNiYuPEVyPseQPjMQ/copy">Tracebase Animal and Sample Table Template</a>.
```

Comment:
NB. I feel like copying when clicking the "Tracebase Animal and Sample Table Template" is a bit unexpected.  A user may only expect to view the template while looking over the procedure, so I think that linking "Copy" may produce a more expected result.  You could additionally link the template without the copy, or add linked "(view)" text.
```suggestion
            <li><a href="https://docs.google.com/spreadsheets/d/1To3495KxJkAtnAD9KVdzc162zKbNiYuPEVyPseQPjMQ/copy">Copy</a> the TraceBase Animal and Sample Table Template.
```